src/Types/Signal.ts

```diff
@@ -14,12 +14,12 @@ type ProcessSenderKeyDistributionMessageOpts = {
 type DecryptSignalProtoOpts = {
 	jid: string
 	type: 'pkmsg' | 'msg'
-	ciphertext: Uint8Array
+	ciphertext: Buffer
```

Comment:
This is needed because libsignal and WAGroup rely heavy on NodeJs Buffer (this will change in the libsignal rewrite too), but the new implementation of binary-wasm rely on pure Uint8Array.

Reply:
This can be more risky and there is the fact that the libsignal repository is not versioned correctly, which can lead to problems in older versions of baileys.

Reply:
> This can be more risky and there is the fact that the libsignal repository is not versioned correctly, which can lead to problems in older versions of baileys.

That's correct, I failed to notice that. We'll make the changes then revert them.